notebooks/lesson04_intro-to-random-forests.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>this then corresponds to the plot above where the graph is rotated such that the red line is on the x-axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>